src/tribler/core/check_os.py

```diff
@@ -45,17 +46,22 @@ def check_environment():
     check_read_write()
 
 
-def check_free_space():
-    logger.info('Check free space')
+def check_free_space(state_dir: Path):
+    """ Check if there is enough free space on the disk."""
+    logger.info(f'Checking free space for the folder: {state_dir}')
```

Comment:
**NIT:** Using f-strings in logging calls is generally discouraged (see for example https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/logging-fstring-interpolation.html).

Reply:
We had this discussion before: #5611 :)

Reply:
Thank you for the suggestion. For this PR, I left the "f-string" version, but we can discuss it at the next dev meeting. In case of an altered agreement, I'll change the formatting for all upcoming pull requests.